awx/main/tasks.py

```diff
@@ -1228,10 +1227,19 @@ def event_handler(self, event_data):
                 event_data = json.loads(event_data_json)
             except json.JSONDecodeError:
                 pass
+        elif isinstance(self, RunJob) and self.instance.gather_event_types in ("none", "errors"):
+            event_level = BasePlaybookEvent._level_event(event_data.get("event"))
+            failed = event_data.get("event") in BasePlaybookEvent.FAILED_EVENTS
+            skip_save = event_level in [0,3]
+            if self.instance.gather_event_types == "errors":
```

Comment:
You type the string "errors" here, but in other places use the variable `JOB_EVENTS_ONLY_ERRORS`. If some places are going to use the hard-coded string, maybe all places should hard-code the string.